draft-yasskin-http-origin-signed-responses.md

```diff
@@ -943,6 +943,11 @@ able to make even one unauthorized signature.
 
 Conforming CAs MUST NOT mark this extension as critical.
 
+A conforming CA MUST NOT issue certificates with this extension unless an
+applicable "issue" or "issuewild" CAA property ({{!RFC6844}}) exists for the CA,
+and the "cansignhttpexchanges" parameter ({{caa-cansignhttpexchanges}}) is
+present on the property and is equal to "yes".
+
```

Comment:
I just realized that this wording could be interpreted as allowing a CA to include the extension if just one of the dNSNames in the certificate has `cansignhttpexchanges=yes`. :grimacing: Obviously, the intent is that this parameter must be present for every dNSName.  I'll propose some better wording in a little bit unless someone else beats me to it.

Reply:
See, this is why I wait. :)

Reply:
How about:

> A conforming CA MUST NOT issue certificates with this extension unless, for each dNSName in the subjectAltName extension of the certificate to be issued:
> 1. An "issue" or "issuewild" CAA property ({{!RFC6844}}) exists that authorizes the CA to issue the certificate; and
> 2. The "cansignhttpexchanges" parameter ({{caa-cansignhttpexchanges}}) is present on the property and is equal to "yes"

Reply:
Done. RFC6844 manages to avoid saying that a certificate's domains are listed in the dNSName, but if saying it here is a problem, we have plenty of time to revise the wording.

Reply:
I cribbed the "each dNSName in the subjectAltName extension" wording from the BR's section on CAA so it should be solid.